src/Lykke.Job.LimitOperationsCollector/Settings/AppSettings.cs

```diff
@@ -1,17 +1,16 @@
 ﻿using System.Collections.Generic;
+using Lykke.Sdk.Settings;
 using Lykke.SettingsReader.Attributes;
 
 namespace Lykke.Job.LimitOperationsCollector.Settings
 {
-    public class AppSettings
+    public class AppSettings : BaseAppSettings
     {
         public LimitOperationsCollectorSettings LimitOperationsCollectorJob { get; set; }
-
-        public SlackNotificationsSettings SlackNotifications { get; set; }
-
-        public MonitoringServiceClientSettings MonitoringServiceClient { get; set; }
-
+        
         public RateCalculatorServiceClient RateCalculatorServiceClient { get; set; }
+
+        public SagasRabbitMqSettings SagasRabbitMq { get; set; }
```

Comment:
Please use the new approved format for such settings.

        public CqrsSettings Cqrs { get; set; }
public class CqrsSettings
    {
        [AmqpCheck]
        public string RabbitConnString { get; set; }
    }

Reply:
not in current release, service uses old settings service, will do during migration